src/main/java/uk/gov/hmcts/dts/fact/mapit/MapItService.java

```diff
@@ -9,6 +10,7 @@
 import uk.gov.hmcts.dts.fact.exception.MapitUsageException;
 
 import java.io.IOException;
+import java.util.Arrays;
 
 @Component
 public class MapItService {
```

Comment:
I have a general comment here. Currently the validation route is like this:
mapit/MapItService -> mapit/MapitValidator -> services/MapitService -> mapit/MapitClient

This is going from the 'mapit' module to the 'services' module and back to the 'mapit' module.

I feel that the 'services' module needs to sit on top and call into the 'mapit' module which then talks to the third party MapIt service. I don't know what you think. It probably needs a bit of discussions. 

I thought the following flow is a bit more natural so the controller will call the 'services' component which then calls the 'mapit' component:
services/validationService -> services/postcodeValidator (or services/postcodeValidationService) -> services/MapitService -> mapit/MapitClient